### ISSUE TITLE
Game log API returns error

### ISSUE THREAD

**Issue by masorz** [coreyjs/nhl-api-py#81]
Game Log for Samuel Helenius is empty:
https://api-web.nhle.com/v1/player/8482726/game-log/20242025/2

Which leads to error: 
```
return self.client.get(resource=f"player/{player_id}/game-log/{season_id}/{game_type}").json()["gameLog"]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'gameLog'

```

```
from nhlpy.nhl_client import NHLClient

regular_season = "2"
samuel_helenius_id = 8482726
season_id = "20242025"

client = NHLClient(verbose=True)

game_log = client.stats.player_game_log(
    player_id=samuel_helenius_id, season_id=season_id, game_type=regular_season
)

```

**Comment by coreyjs** [comment]
Thanks for catching this.  Il get a fix out.

**Comment by coreyjs** [comment]
this was an edge case due to not having any game logs.  Ill have to be safer when extracting from dictionaries.

**Comment by masorz** [comment]
Thanks for doing good work.

**Comment by coreyjs** [comment]
fixed in v 2.12.4
